lib/lotus/generators/application/app.rb

```diff
@@ -110,7 +110,8 @@ def start
           end
 
           unless git_dir_present?
-            cli.template(source.join('gitignore.tt'), target.join('.gitignore'), opts)
+            src_gitignore = if database_type == :file_system then 'gitignore.tt' else '.gitignore' end
```

Comment:
What do you think about changing to a shorter form `database_type == :file_system ? 'blah' : 'ma'`?